<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_submit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>461</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_username"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Name">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_password"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_UserName">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>UserName</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Password">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_confirm_password"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_normal">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Normal</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_vip">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>VIP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_errors">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">fontColor:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_confirm_password</tabstop>
  <tabstop>radioButton_normal</tabstop>
  <tabstop>radioButton_vip</tabstop>
  <tabstop>pushButton_submit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
